src/app/navigation/auth.tsx

```diff
@@ -21,8 +21,8 @@ export const getAuthenticatedGroup = () => {
             header: () => (
               <Header
                 leftText={t('navigation:headers.login')}
-                leftIcon={<BackArrowIcon />}
-                navigation={navigation}
+                headerLeft={<BackArrowIcon />}
+                onPressLeft={() => navigation.goBack()}
```

Comment:
Here you can do this
```suggestion
                onPressLeft={navigation.goBack}
```